### PR TITLE
[excel] (Requirement Set) Update Excel Preview APIs

### PR DESCRIPTION
**Pull request by AlexJerabek**
Fixing the links and updating the content with the latest in Definitely Typed.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[680ee5b](https://github.com/OfficeDev/office-js-docs-pr/commits/680ee5b7cf0eed40cb25223b693898fd3cd08089)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJerabek-patch-1/docs/reference/requirement-sets/excel-preview-apis.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908061935439736-1224%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=aqys7qaFKF0APMEwsD4b0Mxjx7E70dVB9coEAvBS1%2Fc%3D&st=2019-08-06T19%3A31%3A46Z&se=2019-09-06T19%3A36%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/excel-preview-apis.md [hunk 1]
```diff
@@ -362,42 +170,30 @@ The following table lists the Excel JavaScript APIs currently in preview. To see
...
 ||[onColumnSorted](/javascript/api/excel/excel.worksheetcollection#oncolumnsorted)|Occurs when sorting on columns.|
 ||[onFiltered](/javascript/api/excel/excel.worksheetcollection#onfiltered)|Occurs when any worksheet's filter is applied in the workbook.|
 ||[onRowHiddenChanged](/javascript/api/excel/excel.worksheetcollection#onrowhiddenchanged)|Occurs when any worksheet in the workbook has row hidden state changed.|
 ||[onRowSorted](/javascript/api/excel/excel.worksheetcollection#onrowsorted)|Occurs when sorting on rows.|
-||[onSingleClicked](/javascript/api/excel/excel.worksheetcollection#onsingleclicked)|Occurs when left-clicked/tapped operation happens in the worksheet collection.|
+||[onSingleClicked](/javascript/api/excel/excel.worksheetcollection#onsingleclicked)|Occurs when left-clicked/tapped operation happens in the worksheet collection. This event will not be fired when clicking in the following cases:|
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-preview-apis.md]: missing the list of cases?

File: docs/reference/requirement-sets/excel-preview-apis.md [hunk 2]
```diff
@@ -406,7 +202,7 @@ The following table lists the Excel JavaScript APIs currently in preview. To see
 ||[type](/javascript/api/excel/excel.worksheetrowsortedeventargs#type)|Gets the type of the event. See Excel.EventType for details.|
 ||[worksheetId](/javascript/api/excel/excel.worksheetrowsortedeventargs#worksheetid)|Gets the id of the worksheet where the sorting happened.|
 |[WorksheetSingleClickedEventArgs](/javascript/api/excel/excel.worksheetsingleclickedeventargs)|[address](/javascript/api/excel/excel.worksheetsingleclickedeventargs#address)|Gets the address that represents the cell which was left-clicked/tapped for a specific worksheet.|
-||[offsetX](/javascript/api/excel/excel.worksheetsingleclickedeventargs#offsetx)|The distance, in points, from the left-clicked/tapped point to the left (right for RTL) gridline edge of the left-clicked/tapped cell.|
+||[offsetX](/javascript/api/excel/excel.worksheetsingleclickedeventargs#offsetx)|The distance, in points, from the left-clicked/tapped point to the left (or right for RTL) gridline edge of the left-clicked/tapped cell.|
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-preview-apis.md]: I'm not sure that RTL will be clear to users.

File: docs/reference/requirement-sets/excel-preview-apis.md [hunk 3]
```diff
@@ -362,42 +170,30 @@ The following table lists the Excel JavaScript APIs currently in preview. To see
...
 |[Worksheet](/javascript/api/excel/excel.worksheet)|[comments](/javascript/api/excel/excel.worksheet#comments)|Returns a collection of all the Comments objects on the worksheet. Read-only.|
 ||[onColumnSorted](/javascript/api/excel/excel.worksheet#oncolumnsorted)|Occurs when sorting on columns.|
 ||[onFiltered](/javascript/api/excel/excel.worksheet#onfiltered)|Occurs when filter is applied on a specific worksheet.|
 ||[onRowHiddenChanged](/javascript/api/excel/excel.worksheet#onrowhiddenchanged)|Occurs when row hidden state changed on a specific worksheet.|
 ||[onRowSorted](/javascript/api/excel/excel.worksheet#onrowsorted)|Occurs when sorting on rows.|
-||[onSingleClicked](/javascript/api/excel/excel.worksheet#onsingleclicked)|Occurs when left-clicked/tapped operation happens in the worksheet.|
+||[onSingleClicked](/javascript/api/excel/excel.worksheet#onsingleclicked)|Occurs when left-clicked/tapped operation happens in the worksheet. This event will not be fired when clicking in the following cases:|
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-preview-apis.md]: missing the list of cases?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9668b94](https://github.com/OfficeDev/office-js-docs-pr/commits/9668b943095d6ac735d1b3cb77a234a2866b2a4e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJerabek-patch-1/docs/reference/requirement-sets/excel-preview-apis.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908071906547344-1224%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Fug1MbEB5NdAtsrDop21GexclAUIyc4%2FsLHDxAC%2BO3U%3D&st=2019-08-07T19%3A03%3A04Z&se=2019-09-07T19%3A08%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT The script cuts off descriptions after the first line. I added a `[...]` to each to make it clear the longer description is after link.

File: docs/reference/requirement-sets/excel-preview-apis.md [hunk 4]
```diff
@@ -406,7 +202,7 @@ The following table lists the Excel JavaScript APIs currently in preview. To see
 ||[type](/javascript/api/excel/excel.worksheetrowsortedeventargs#type)|Gets the type of the event. See Excel.EventType for details.|
 ||[worksheetId](/javascript/api/excel/excel.worksheetrowsortedeventargs#worksheetid)|Gets the id of the worksheet where the sorting happened.|
 |[WorksheetSingleClickedEventArgs](/javascript/api/excel/excel.worksheetsingleclickedeventargs)|[address](/javascript/api/excel/excel.worksheetsingleclickedeventargs#address)|Gets the address that represents the cell which was left-clicked/tapped for a specific worksheet.|
-||[offsetX](/javascript/api/excel/excel.worksheetsingleclickedeventargs#offsetx)|The distance, in points, from the left-clicked/tapped point to the left (right for RTL) gridline edge of the left-clicked/tapped cell.|
+||[offsetX](/javascript/api/excel/excel.worksheetsingleclickedeventargs#offsetx)|The distance, in points, from the left-clicked/tapped point to the left (or right for right-to-left documents) gridline edge of the left-clicked/tapped cell.|
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-preview-apis.md]:
```suggestion
||[offsetX](/javascript/api/excel/excel.worksheetsingleclickedeventargs#offsetx)|The distance, in points, from the left-clicked/tapped point to the left (or right for right-to-left languages) gridline edge of the left-clicked/tapped cell.|
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[baa9f99](https://github.com/OfficeDev/office-js-docs-pr/commits/baa9f9934e3ad38614fbfa9af88598f8d3daafc4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJerabek-patch-1/docs/reference/requirement-sets/excel-preview-apis.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908072030058026-1224%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=wDm89lIOOwyQCSCbHq6CmMpPCd9DVqRx77XWmOqW8NM%3D&st=2019-08-07T20%3A26%3A15Z&se=2019-09-07T20%3A31%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**